.github/workflows/turing.yaml

```diff
@@ -342,15 +342,22 @@ jobs:
       KNATIVE_VERSION: v0.18.3
       KNATIVE_ISTIO_VERSION: v0.18.1
       LOCAL_REGISTRY: registry.localhost:5000
-      MLP_VERSION: v1.4.14
-      MERLIN_VERSION: v0.9.0
-      VAULT_CHART_VERSION: 0.16.1
       TURING_API_VERSION: ${{ needs.build-api.outputs.api-version }}
       TURING_ROUTER_VERSION: ${{ needs.build-router.outputs.router-version }}
     needs:
       - build-api
       - build-router
       - build-cluster-init
+    strategy:
+      fail-fast: false
+      matrix:
+        useInClusterConfig: [false, true]
```

Comment:
![Screenshot 2021-11-15 at 9 57 47 AM](https://user-images.githubusercontent.com/13537118/141710464-0ecbc335-026a-49ee-9cc1-e2fbc0198c39.png)

I'm not sure if it's possible but can we have nicer names other than true, false here?

Reply:
This is how it works by design, but I have some workaround in mind, let me try it.

Reply:
Better now? 
<img width="373" alt="Screenshot 2021-11-15 at 10 17 03 AM" src="https://user-images.githubusercontent.com/1886194/141712377-d4a5246c-fb89-486e-a638-2f6bad1f059e.png">

Reply:
Yah, saw this change. Nice.